mempool/v0/reactor_test.go

```diff
@@ -119,9 +121,6 @@ func TestReactorConcurrency(t *testing.T) {
 			err := reactors[1].mempool.Update(1, []types.Tx{}, make([]*abci.ResponseDeliverTx, 0), nil, nil)
 			assert.NoError(t, err)
 		}()
-
-		// 1. flush the mempool
-		reactors[1].mempool.Flush()
```

Comment:
this is the flush that causes a weird race issue where the mempool gets cleared while processing transactions